From: Stefan Bader <stefan.bader@canonical.com>
Date: Wed, 9 Apr 2008 20:35:12 +0000 (-0400)
Subject: UBUNTU: SAUCE: mmc: Increase power_up deleay to fix TI readers
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=7fd1cd92f263a1891e543b16b3edc64c618dcee9

UBUNTU: SAUCE: mmc: Increase power_up deleay to fix TI readers
OriginalAuthor: Pascal Terjan <pterjan@mandriva.com>
Bug: #137686

Signed-off-by: Pascal Terjan <pterjan@mandriva.com>
Signed-off-by: Stefan Bader <stefan.bader@canonical.com>
---

Index: 2008-devel/drivers/mmc/core/core.c
===================================================================
--- 2008-devel.orig/drivers/mmc/core/core.c
+++ 2008-devel/drivers/mmc/core/core.c
@@ -496,7 +496,7 @@ static void mmc_power_up(struct mmc_host
 	 * This delay must be at least 74 clock sizes, or 1 ms, or the
 	 * time required to reach a stable voltage.
 	 */
-	mmc_delay(2);
+	mmc_delay(10);
 }
 
 static void mmc_power_off(struct mmc_host *host)
